client/src/components/atoms/CommentEditorButton.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <q-btn
+    :aria-label="$t(props.ariaLabel)"
+    color="black"
+    outline
+    dense
+    size="sm"
+    :class="{ 'is-active': props.isActive }"
+    :disabled="props.isDisabled"
+    @click="props.clickHandler"
+  >
+    <q-tooltip
+      class="bg-primary"
+      anchor="top middle"
+      self="bottom middle"
+      :offset="[0, 5]"
+      >{{ $t(props.tooltipText) }}</q-tooltip
```

Comment:
```suggestion
      >{{ props.tooltipText }}</q-tooltip
```
I'd strongly suggest sending in translated string rather than translation keys for consistency.  This also removes the dependency on useI18n in the component making it easier to test

I'd recommend using i18n in the parent component and sending in a translated string.

Reply:
I placed this in the child rather than the parent because it made the code more DRY. 

I'll refactor as suggested.

Reply:
Yeah, I can totally appreciate that. It’s a give and take between dependencies and dry in this case probably. 

Another thought, if we’re going to do this, we should perhaps use a specifically named prop (tAriaLabel maybe?) for future clarity’s sake?

Reply:
This has been refactored.